src/os/filestore/FileJournal.cc

```diff
@@ -1217,9 +1217,11 @@ void FileJournal::write_thread_entry()
       // but should be fine given that we will have plenty of aios in
       // flight if we hit this limit to ensure we keep the device
       // saturated.
+      uint64_t min_size = 64 * 1024;
       while (aio_num > 0) {
 	int exp = MIN(aio_num * 2, 24);
 	long unsigned min_new = 1ull << exp;
+	min_new = min_new < min_size ? min_size : min_new;
```

Comment:
@liewegas  I think i mis-understand the logic of those code. For my example:
io=1,aio_write_queue_bytes=8k
after submit: aio=1, aio_write_queue_bytes=0
Then next entry: len=8k-->aio_write_queue_bytes=8k.
it must wait because 1 <<(MIN(aio *2, 24)) = 4 < aio_write_queue_bytes. --->for this case,it dont wait. Because the condition is cur>= min_new that is:8k >=4. so break. and do continue.

But i rethink the code again. Those code to limit more aio in flight. Is it ok? If ok, Consider this case
aio=10; min_new=1<<(10*2)=1M; 
if cur=1M then cur>= min_size, it can continue. add more aios.
If i understand correctly, i think the judgement is like 'if cur + aio_in_flight < max; break'.
Is it correctly?